tests/unit/test_session.py

```diff
@@ -231,3 +232,33 @@ def test_explain_query_error():
     session._run_query = MagicMock()
     session._run_query.side_effect = ProgrammingError("Can't explain.")
     assert session._explain_query("select 1") is None
+
+
+def test_parse_table_name():
```

Comment:
Could you:
- add some negative tests?
- have tests with different cases of letters and special letters?

Reply:
updated